docs/WritingTests.asciidoc

```diff
@@ -265,20 +265,25 @@ In short, writing multi-machine tests adds a few more layers of complexity:
 
 ==== Job dependencies
 
-There are 2 types of dependencies: +CHAINED+ and +PARALLEL+:
+There are 3 types of dependencies: +CHAINED+, +DIRECTLY_CHAINED+ and +PARALLEL+:
 
-* +CHAINED+ describes when one test case depends on another and both are run sequentially, i.e. KDE test suite is run after and only after Installation test suite
+* +CHAINED+ and +DIRECTLY_CHAINED+ describe when one test case depends on another and both are run sequentially, i.e. KDE test suite is run after and only after Installation test suite
 is successfully finished and cancelled if fail.
 
-To define +CHAINED+ dependency add variable +START_AFTER_TEST+ with the name(s) of test suite(s) after which selected test suite is supposed to run.
+The difference between +CHAINED+ and +DIRECTLY_CHAINED+ dependencies is that +DIRECTLY_CHAINED+ means the tests must run directly after another on the same worker slot.
```

Comment:
I'd explain the context, why `DIRECTLY_CHAINED` is needed. Just a note that it's needed for IPMI or other backends, which does not produce qcow2 image.

Reply:
I did not mention IPMI directly because I find it very specific. But I added a more general note.